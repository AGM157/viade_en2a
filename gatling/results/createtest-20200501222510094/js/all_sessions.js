allUsersData = {
    
color: '#FF9D00',
name: 'Active Users',
data: [
  [1588371913000,10],[1588371914000,10],[1588371915000,10],[1588371916000,10],[1588371917000,10],[1588371918000,10],[1588371919000,10],[1588371920000,10],[1588371921000,10],[1588371922000,10],[1588371923000,10],[1588371924000,10],[1588371925000,10],[1588371926000,10],[1588371927000,10],[1588371928000,10],[1588371929000,10],[1588371930000,10],[1588371931000,10],[1588371932000,10],[1588371933000,10],[1588371934000,10],[1588371935000,10],[1588371936000,10],[1588371937000,10],[1588371938000,10],[1588371939000,10],[1588371940000,10],[1588371941000,10],[1588371942000,10],[1588371943000,10],[1588371944000,10],[1588371945000,10],[1588371946000,10],[1588371947000,10],[1588371948000,10],[1588371949000,10],[1588371950000,10],[1588371951000,10],[1588371952000,10],[1588371953000,10],[1588371954000,10],[1588371955000,10],[1588371956000,10],[1588371957000,10],[1588371958000,10],[1588371959000,10],[1588371960000,10],[1588371961000,10],[1588371962000,10],[1588371963000,10],[1588371964000,10],[1588371965000,10],[1588371966000,9],[1588371967000,9],[1588371968000,7],[1588371969000,7],[1588371970000,6],[1588371971000,6],[1588371972000,5],[1588371973000,5],[1588371974000,5],[1588371975000,5],[1588371976000,5],[1588371977000,5],[1588371978000,5],[1588371979000,5],[1588371980000,5],[1588371981000,5],[1588371982000,5],[1588371983000,5],[1588371984000,5],[1588371985000,5],[1588371986000,5],[1588371987000,5],[1588371988000,4],[1588371989000,4],[1588371990000,3],[1588371991000,3],[1588371992000,2],[1588371993000,2],[1588371994000,2],[1588371995000,2],[1588371996000,2],[1588371997000,2],[1588371998000,2],[1588371999000,2],[1588372000000,2],[1588372001000,2],[1588372002000,2],[1588372003000,2],[1588372004000,2],[1588372005000,1],[1588372006000,1],[1588372007000,1],[1588372008000,1],[1588372009000,1],[1588372010000,1],[1588372011000,1],[1588372012000,1],[1588372013000,1]
],
tooltip: { yDecimals: 0, ySuffix: '', valueDecimals: 0 }
    , zIndex: 20
    , yAxis: 1
};