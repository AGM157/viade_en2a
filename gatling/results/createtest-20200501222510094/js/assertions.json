{
  "simulation": "viadeGatling.CreateTest",
  "simulationId": "createtest",
  "start": 1588371910094,
  "description": "",
  "scenarios": ["CreateTest"],
  "assertions": [

  ]
}