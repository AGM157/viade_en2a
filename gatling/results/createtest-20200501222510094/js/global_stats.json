{
    "name": "Global Information",
    "numberOfRequests": {
        "total": 943,
        "ok": 902,
        "ko": 41
    },
    "minResponseTime": {
        "total": 1,
        "ok": 1,
        "ko": 197
    },
    "maxResponseTime": {
        "total": 60001,
        "ok": 34424,
        "ko": 60001
    },
    "meanResponseTime": {
        "total": 909,
        "ok": 858,
        "ko": 2025
    },
    "standardDeviation": {
        "total": 3186,
        "ok": 2593,
        "ko": 9177
    },
    "percentiles1": {
        "total": 205,
        "ok": 192,
        "ko": 389
    },
    "percentiles2": {
        "total": 586,
        "ok": 588,
        "ko": 514
    },
    "percentiles3": {
        "total": 3357,
        "ok": 3362,
        "ko": 1562
    },
    "percentiles4": {
        "total": 8782,
        "ok": 7909,
        "ko": 36667
    },
    "group1": {
    "name": "t < 800 ms",
    "count": 730,
    "percentage": 77
},
    "group2": {
    "name": "800 ms < t < 1200 ms",
    "count": 18,
    "percentage": 2
},
    "group3": {
    "name": "t > 1200 ms",
    "count": 154,
    "percentage": 16
},
    "group4": {
    "name": "failed",
    "count": 41,
    "percentage": 4
},
    "meanNumberOfRequestsPerSecond": {
        "total": 9.336633663366337,
        "ok": 8.930693069306932,
        "ko": 0.40594059405940597
    }
}