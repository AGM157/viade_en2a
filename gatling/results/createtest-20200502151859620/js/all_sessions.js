allUsersData = {
    
color: '#FF9D00',
name: 'Active Users',
data: [
  [1588432742000,628],[1588432743000,1000],[1588432744000,1000],[1588432745000,1000],[1588432746000,1000],[1588432747000,1000],[1588432748000,1000],[1588432749000,1000],[1588432750000,1000],[1588432751000,1000],[1588432752000,1000],[1588432753000,1000],[1588432754000,1000],[1588432755000,1000],[1588432756000,1000],[1588432757000,1000],[1588432758000,1000],[1588432759000,1000],[1588432760000,1000],[1588432761000,1000],[1588432762000,1000],[1588432763000,1000],[1588432764000,1000],[1588432765000,1000],[1588432766000,1000],[1588432767000,1000],[1588432768000,1000],[1588432769000,1000],[1588432770000,1000],[1588432771000,1000],[1588432772000,1000],[1588432773000,1000],[1588432774000,1000],[1588432775000,1000],[1588432776000,1000],[1588432777000,1000],[1588432778000,1000],[1588432779000,1000],[1588432780000,1000],[1588432781000,1000],[1588432782000,1000],[1588432783000,1000],[1588432784000,1000],[1588432785000,1000],[1588432786000,1000],[1588432787000,1000],[1588432788000,1000],[1588432789000,1000],[1588432790000,1000],[1588432791000,1000],[1588432792000,1000],[1588432793000,1000],[1588432794000,1000],[1588432795000,1000],[1588432796000,1000],[1588432797000,1000],[1588432798000,1000],[1588432799000,1000],[1588432800000,1000],[1588432801000,1000],[1588432802000,1000],[1588432803000,1000],[1588432804000,1000],[1588432805000,1000],[1588432806000,1000],[1588432807000,1000],[1588432808000,1000],[1588432809000,1000],[1588432810000,1000],[1588432811000,1000],[1588432812000,1000],[1588432813000,1000],[1588432814000,1000],[1588432815000,1000],[1588432816000,1000],[1588432817000,1000],[1588432818000,1000],[1588432819000,1000],[1588432820000,1000],[1588432821000,1000],[1588432822000,1000],[1588432823000,1000],[1588432824000,1000],[1588432825000,1000],[1588432826000,1000],[1588432827000,1000],[1588432828000,1000],[1588432829000,1000],[1588432830000,1000],[1588432831000,1000],[1588432832000,1000],[1588432833000,1000],[1588432834000,1000],[1588432835000,1000],[1588432836000,1000],[1588432837000,1000],[1588432838000,1000],[1588432839000,1000],[1588432840000,1000],[1588432841000,1000],[1588432842000,1000],[1588432843000,1000],[1588432844000,1000],[1588432845000,1000],[1588432846000,1000],[1588432847000,1000],[1588432848000,1000],[1588432849000,1000],[1588432850000,1000],[1588432851000,1000],[1588432852000,1000],[1588432853000,1000],[1588432854000,1000],[1588432855000,1000],[1588432856000,1000],[1588432857000,1000],[1588432858000,1000],[1588432859000,1000],[1588432860000,1000],[1588432861000,1000],[1588432862000,1000],[1588432863000,1000],[1588432864000,1000],[1588432865000,1000],[1588432866000,1000],[1588432867000,1000],[1588432868000,1000],[1588432869000,1000],[1588432870000,1000],[1588432871000,1000],[1588432872000,1000],[1588432873000,1000],[1588432874000,1000],[1588432875000,993],[1588432876000,976],[1588432877000,953],[1588432878000,953],[1588432879000,953],[1588432880000,899],[1588432881000,892],[1588432882000,891],[1588432883000,890],[1588432884000,890],[1588432885000,877],[1588432886000,874],[1588432887000,874],[1588432888000,873],[1588432889000,851],[1588432890000,843],[1588432891000,831],[1588432892000,831],[1588432893000,831],[1588432894000,746],[1588432895000,746],[1588432896000,669],[1588432897000,669],[1588432898000,636],[1588432899000,634],[1588432900000,633],[1588432901000,632],[1588432902000,632],[1588432903000,632],[1588432904000,632],[1588432905000,632],[1588432906000,632],[1588432907000,632],[1588432908000,632],[1588432909000,632],[1588432910000,632],[1588432911000,632],[1588432912000,632],[1588432913000,631],[1588432914000,631],[1588432915000,631],[1588432916000,624],[1588432917000,590],[1588432918000,547],[1588432919000,487],[1588432920000,423],[1588432921000,423],[1588432922000,423],[1588432923000,423],[1588432924000,423],[1588432925000,423],[1588432926000,423],[1588432927000,423],[1588432928000,423],[1588432929000,423],[1588432930000,423],[1588432931000,423],[1588432932000,423],[1588432933000,423],[1588432934000,423],[1588432935000,423],[1588432936000,423],[1588432937000,423],[1588432938000,423],[1588432939000,423],[1588432940000,423],[1588432941000,423],[1588432942000,423],[1588432943000,422],[1588432944000,422],[1588432945000,422],[1588432946000,422],[1588432947000,422],[1588432948000,422],[1588432949000,422],[1588432950000,422],[1588432951000,422],[1588432952000,422],[1588432953000,422],[1588432954000,422],[1588432955000,422],[1588432956000,422],[1588432957000,422],[1588432958000,422],[1588432959000,422],[1588432960000,422],[1588432961000,422],[1588432962000,422],[1588432963000,422],[1588432964000,422],[1588432965000,422],[1588432966000,422],[1588432967000,422],[1588432968000,422],[1588432969000,422],[1588432970000,422],[1588432971000,422],[1588432972000,422],[1588432973000,422],[1588432974000,422],[1588432975000,422],[1588432976000,422],[1588432977000,422],[1588432978000,422],[1588432979000,422],[1588432980000,422],[1588432981000,422],[1588432982000,422],[1588432983000,422],[1588432984000,422],[1588432985000,422],[1588432986000,422],[1588432987000,422],[1588432988000,422],[1588432989000,420],[1588432990000,420],[1588432991000,420],[1588432992000,420],[1588432993000,420],[1588432994000,420],[1588432995000,420],[1588432996000,420],[1588432997000,420],[1588432998000,420],[1588432999000,420],[1588433000000,420],[1588433001000,420],[1588433002000,420],[1588433003000,420],[1588433004000,420],[1588433005000,420],[1588433006000,420],[1588433007000,420],[1588433008000,420],[1588433009000,420],[1588433010000,420],[1588433011000,420],[1588433012000,420],[1588433013000,420],[1588433014000,420],[1588433015000,420],[1588433016000,420],[1588433017000,420],[1588433018000,420],[1588433019000,420],[1588433020000,420],[1588433021000,420],[1588433022000,420],[1588433023000,419],[1588433024000,419],[1588433025000,419],[1588433026000,419],[1588433027000,419],[1588433028000,419],[1588433029000,419],[1588433030000,419],[1588433031000,419],[1588433032000,419],[1588433033000,419],[1588433034000,419],[1588433035000,419],[1588433036000,419],[1588433037000,419],[1588433038000,419],[1588433039000,419],[1588433040000,419],[1588433041000,419],[1588433042000,419],[1588433043000,419],[1588433044000,419],[1588433045000,419],[1588433046000,419],[1588433047000,419],[1588433048000,419],[1588433049000,419],[1588433050000,419],[1588433051000,419],[1588433052000,419],[1588433053000,419],[1588433054000,419],[1588433055000,419],[1588433056000,419],[1588433057000,419],[1588433058000,419],[1588433059000,419],[1588433060000,419],[1588433061000,419],[1588433062000,419],[1588433063000,419],[1588433064000,419],[1588433065000,419],[1588433066000,419],[1588433067000,419],[1588433068000,419],[1588433069000,419],[1588433070000,419],[1588433071000,419],[1588433072000,419],[1588433073000,419],[1588433074000,419],[1588433075000,419],[1588433076000,419],[1588433077000,419],[1588433078000,419],[1588433079000,419],[1588433080000,419],[1588433081000,418],[1588433082000,418],[1588433083000,418],[1588433084000,417],[1588433085000,417],[1588433086000,417],[1588433087000,417],[1588433088000,417],[1588433089000,417],[1588433090000,416],[1588433091000,416],[1588433092000,416],[1588433093000,416],[1588433094000,416],[1588433095000,416],[1588433096000,416],[1588433097000,415],[1588433098000,415],[1588433099000,415],[1588433100000,415],[1588433101000,415],[1588433102000,415],[1588433103000,394],[1588433104000,394],[1588433105000,392],[1588433106000,392],[1588433107000,392],[1588433108000,392],[1588433109000,392],[1588433110000,392],[1588433111000,392],[1588433112000,392],[1588433113000,392],[1588433114000,388],[1588433115000,388],[1588433116000,376],[1588433117000,376],[1588433118000,367],[1588433119000,335],[1588433120000,335],[1588433121000,331],[1588433122000,331],[1588433123000,316],[1588433124000,316],[1588433125000,300],[1588433126000,300],[1588433127000,300],[1588433128000,279],[1588433129000,279],[1588433130000,279],[1588433131000,279],[1588433132000,279],[1588433133000,278],[1588433134000,277],[1588433135000,277],[1588433136000,277],[1588433137000,277],[1588433138000,277],[1588433139000,277],[1588433140000,276],[1588433141000,276],[1588433142000,276],[1588433143000,275],[1588433144000,275],[1588433145000,274],[1588433146000,274],[1588433147000,274],[1588433148000,273],[1588433149000,273],[1588433150000,273],[1588433151000,273],[1588433152000,273],[1588433153000,273],[1588433154000,272],[1588433155000,272],[1588433156000,272],[1588433157000,256],[1588433158000,255],[1588433159000,249],[1588433160000,248],[1588433161000,247],[1588433162000,243],[1588433163000,243],[1588433164000,243],[1588433165000,229],[1588433166000,225],[1588433167000,224],[1588433168000,219],[1588433169000,201],[1588433170000,199],[1588433171000,199],[1588433172000,196],[1588433173000,196],[1588433174000,194],[1588433175000,194],[1588433176000,192],[1588433177000,192],[1588433178000,191],[1588433179000,191],[1588433180000,166],[1588433181000,166],[1588433182000,148],[1588433183000,102],[1588433184000,101],[1588433185000,100],[1588433186000,97],[1588433187000,97],[1588433188000,97],[1588433189000,97],[1588433190000,97],[1588433191000,97],[1588433192000,87],[1588433193000,78],[1588433194000,78],[1588433195000,78],[1588433196000,76],[1588433197000,76],[1588433198000,33],[1588433199000,28],[1588433200000,24],[1588433201000,24],[1588433202000,24],[1588433203000,24],[1588433204000,24],[1588433205000,24],[1588433206000,24],[1588433207000,24],[1588433208000,24],[1588433209000,24],[1588433210000,23]
],
tooltip: { yDecimals: 0, ySuffix: '', valueDecimals: 0 }
    , zIndex: 20
    , yAxis: 1
};