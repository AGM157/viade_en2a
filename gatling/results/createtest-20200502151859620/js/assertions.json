{
  "simulation": "viadeGatling.CreateTest",
  "simulationId": "createtest",
  "start": 1588432739620,
  "description": "",
  "scenarios": ["CreateTest"],
  "assertions": [

  ]
}