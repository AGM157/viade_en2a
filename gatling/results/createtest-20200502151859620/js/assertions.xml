<testsuite name="viadeGatling.CreateTest" tests="0" errors="0" failures="0" time="0">

</testsuite>