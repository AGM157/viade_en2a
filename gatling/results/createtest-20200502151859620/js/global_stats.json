{
    "name": "Global Information",
    "numberOfRequests": {
        "total": 33967,
        "ok": 22860,
        "ko": 11107
    },
    "minResponseTime": {
        "total": 1,
        "ok": 1,
        "ko": 406
    },
    "maxResponseTime": {
        "total": 60877,
        "ok": 60011,
        "ko": 60877
    },
    "meanResponseTime": {
        "total": 18791,
        "ok": 9692,
        "ko": 37519
    },
    "standardDeviation": {
        "total": 20707,
        "ok": 11944,
        "ko": 22284
    },
    "percentiles1": {
        "total": 9383,
        "ok": 5892,
        "ko": 29617
    },
    "percentiles2": {
        "total": 24332,
        "ok": 9985,
        "ko": 60003
    },
    "percentiles3": {
        "total": 60014,
        "ok": 37435,
        "ko": 60078
    },
    "percentiles4": {
        "total": 60099,
        "ok": 55012,
        "ko": 60214
    },
    "group1": {
    "name": "t < 800 ms",
    "count": 3364,
    "percentage": 10
},
    "group2": {
    "name": "800 ms < t < 1200 ms",
    "count": 732,
    "percentage": 2
},
    "group3": {
    "name": "t > 1200 ms",
    "count": 18764,
    "percentage": 55
},
    "group4": {
    "name": "failed",
    "count": 11107,
    "percentage": 33
},
    "meanNumberOfRequestsPerSecond": {
        "total": 72.42430703624734,
        "ok": 48.74200426439232,
        "ko": 23.68230277185501
    }
}