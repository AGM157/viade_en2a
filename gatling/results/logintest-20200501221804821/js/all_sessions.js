allUsersData = {
    
color: '#FF9D00',
name: 'Active Users',
data: [
  [1588371487000,100],[1588371488000,100],[1588371489000,100],[1588371490000,100],[1588371491000,100],[1588371492000,100],[1588371493000,100],[1588371494000,100],[1588371495000,100],[1588371496000,100],[1588371497000,100],[1588371498000,100],[1588371499000,100],[1588371500000,100],[1588371501000,100],[1588371502000,100],[1588371503000,100],[1588371504000,100],[1588371505000,100],[1588371506000,100],[1588371507000,100],[1588371508000,100],[1588371509000,100],[1588371510000,100],[1588371511000,100],[1588371512000,100],[1588371513000,100],[1588371514000,100],[1588371515000,100],[1588371516000,100],[1588371517000,100],[1588371518000,100],[1588371519000,100],[1588371520000,100],[1588371521000,100],[1588371522000,100],[1588371523000,100],[1588371524000,100],[1588371525000,100],[1588371526000,99],[1588371527000,97],[1588371528000,96],[1588371529000,95],[1588371530000,93],[1588371531000,92],[1588371532000,91],[1588371533000,91],[1588371534000,91],[1588371535000,91],[1588371536000,89],[1588371537000,88],[1588371538000,88],[1588371539000,88],[1588371540000,86],[1588371541000,86],[1588371542000,86],[1588371543000,86],[1588371544000,86],[1588371545000,86],[1588371546000,86],[1588371547000,86],[1588371548000,85],[1588371549000,84],[1588371550000,84],[1588371551000,82],[1588371552000,81],[1588371553000,81],[1588371554000,81],[1588371555000,81],[1588371556000,81],[1588371557000,79],[1588371558000,79],[1588371559000,79],[1588371560000,79],[1588371561000,79],[1588371562000,79],[1588371563000,79],[1588371564000,79],[1588371565000,79],[1588371566000,77],[1588371567000,75],[1588371568000,69],[1588371569000,62],[1588371570000,61],[1588371571000,50],[1588371572000,50],[1588371573000,50],[1588371574000,50],[1588371575000,50]
],
tooltip: { yDecimals: 0, ySuffix: '', valueDecimals: 0 }
    , zIndex: 20
    , yAxis: 1
};