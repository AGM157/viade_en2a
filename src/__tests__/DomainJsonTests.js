// jest-dom adds custom jest matchers for asserting on DOM nodes.
// allows you to do things like:
// expect(element).toHaveTextContent(/react/i)
// learn more: https://github.com/testing-library/jest-dom
import '@testing-library/jest-dom/';
import parseRouteJsonLD from '../services/importing/DomainJSONLDParser.js'
import {TrackPoint, Resource, Comment, Route} from "../domain/DomainClasses.js"


const comments = [
    new Comment("http://inrupt.luispc1998/routeComments/<comentarioConSuTipo>"),
    new Comment("http://inrupt.angelixus/routeComments/<comentarioConSuTipo>"),
    new Comment("http://inrupt.raupemol/routeComments/<comentarioConSuTipo>"),
    new Comment("http://inrupt.luispc1998/routeComments/<comentarioConSuTipo>")
]


const media = [
    new Resource("http://inrupt.luispc1998/routeMedia/image/<comentarioConSuTipo>"),
    new Resource("http://inrupt.angelixus/routeMedia/image/<comentarioConSuTipo>"),
    new Resource("http://inrupt.raupemol/routeMedia/video/<comentarioConSuTipo>"),
    new Resource("http://inrupt.luispc1998/routeMedia/video/<comentarioConSuTipo>")
]

const points = [
    new TrackPoint(45.123, 34.121),
    new TrackPoint(46.123, 34.121),
    new TrackPoint(47.123, 34.121),
    new TrackPoint(48.123, 32.121),
    new TrackPoint(49.123, 34.121),
    new TrackPoint(40.123, 32.121),
    new TrackPoint(50.123, 33.121),
    new TrackPoint(53.123, 34.121),
    new TrackPoint(54.123, 34.121),
    new TrackPoint(55.123, 35.121),
    new TrackPoint(55.123, 34.121)
]

const params = {
    name:"Route test 1",
    description:"This is a test to see the output of the JsonLDConversor",
    itinerary:points,
    comments:comments,
    resources:media

}

const route = new Route(params)



test("Parsing example data", () =>{
    expect(parseRouteJsonLD(route)).toMatchSnapshot()
})



const media2 = [
    new Resource("http://inrupt.luispc1998/routeMedia/image/<comentarioConSuTipo>"),
    new Resource("http://inrupt.angelixus/routeMedia/image/<comentarioConSuTipo>"),
    new Resource("http://inrupt.raupemol/routeMedia/video/<comentarioConSuTipo>"),
    new Resource("http://inrupt.luispc1998/routeMedia/video/<comentarioConSuTipo>"),
    new Resource("http://inrupt.luispc1998/routeMedia/video/<comentarioConSuTipo>"),
    new Resource("http://inrupt.luispc1998/routeMedia/video/<comentarioConSuTipo>")
]

const points2 = [
    new TrackPoint(45.123, 34.121),
    new TrackPoint(46.123, 34.121),
    new TrackPoint(47.123, 34.121),
    new TrackPoint(48.123, 32.121),
    new TrackPoint(49.123, 34.121),
    new TrackPoint(40.123, 32.121),
    new TrackPoint(50.123, 33.121),
    new TrackPoint(53.123, 34.121),
    new TrackPoint(54.123, 34.121),
    new TrackPoint(55.123, 35.121),
    new TrackPoint(57.123, 30.121),
    new TrackPoint(56.123, 26.121),
    new TrackPoint(53.123, 25.121),
    new TrackPoint(55.123, 20.121),
    new TrackPoint(51.123, 15.121)
]

const params2 = {
    name:"Route test 2",
    description:"This is a test to see the output of the JsonLDConversor",
    itinerary:points2,
    comments:comments,
    resources:media2

}


const route2 = new Route(params2)



test("Parsing example data2", () =>{
    expect(parseRouteJsonLD(route2)).toMatchSnapshot()
})