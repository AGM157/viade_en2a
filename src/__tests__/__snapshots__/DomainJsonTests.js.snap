// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parsing example data 1`] = `
Object {
  "@context": Object {
    "@version": 1.1,
    "comments": Object {
      "@container": "@list",
      "@id": "viade:comments",
    },
    "description": Object {
      "@id": "schema:description",
      "@type": "xs:string",
    },
    "latitude": Object {
      "@id": "schema:longitude",
      "@type": "xs:double",
    },
    "media": Object {
      "@container": "@list",
      "@id": "viade:media",
    },
    "name": Object {
      "@id": "schema:name",
      "@type": "xs:string",
    },
    "points": Object {
      "@container": "@list",
      "@id": "viade:points",
    },
    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
    "schema": "http://schema.org/",
    "viade": "http://arquisoft.github.io/viadeSpec/",
    "xsd": "http://www.w3.org/2001/XMLSchema#",
  },
  "comments": Array [
    Object {
      "@id": "http://inrupt.luispc1998/routeComments/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.angelixus/routeComments/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.raupemol/routeComments/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.luispc1998/routeComments/<comentarioConSuTipo>",
    },
  ],
  "description": "This is a test to see the output of the JsonLDConversor",
  "media": Array [
    Object {
      "@id": "http://inrupt.luispc1998/routeMedia/image/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.angelixus/routeMedia/image/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.raupemol/routeMedia/video/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.luispc1998/routeMedia/video/<comentarioConSuTipo>",
    },
  ],
  "name": "Route test 1",
  "points": Array [
    Object {
      "latitude": 45.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 46.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 47.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 48.123,
      "longitude": 32.121,
    },
    Object {
      "latitude": 49.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 40.123,
      "longitude": 32.121,
    },
    Object {
      "latitude": 50.123,
      "longitude": 33.121,
    },
    Object {
      "latitude": 53.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 54.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 55.123,
      "longitude": 35.121,
    },
    Object {
      "latitude": 55.123,
      "longitude": 34.121,
    },
  ],
}
`;

exports[`Parsing example data2 1`] = `
Object {
  "@context": Object {
    "@version": 1.1,
    "comments": Object {
      "@container": "@list",
      "@id": "viade:comments",
    },
    "description": Object {
      "@id": "schema:description",
      "@type": "xs:string",
    },
    "latitude": Object {
      "@id": "schema:longitude",
      "@type": "xs:double",
    },
    "media": Object {
      "@container": "@list",
      "@id": "viade:media",
    },
    "name": Object {
      "@id": "schema:name",
      "@type": "xs:string",
    },
    "points": Object {
      "@container": "@list",
      "@id": "viade:points",
    },
    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
    "schema": "http://schema.org/",
    "viade": "http://arquisoft.github.io/viadeSpec/",
    "xsd": "http://www.w3.org/2001/XMLSchema#",
  },
  "comments": Array [
    Object {
      "@id": "http://inrupt.luispc1998/routeComments/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.angelixus/routeComments/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.raupemol/routeComments/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.luispc1998/routeComments/<comentarioConSuTipo>",
    },
  ],
  "description": "This is a test to see the output of the JsonLDConversor",
  "media": Array [
    Object {
      "@id": "http://inrupt.luispc1998/routeMedia/image/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.angelixus/routeMedia/image/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.raupemol/routeMedia/video/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.luispc1998/routeMedia/video/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.luispc1998/routeMedia/video/<comentarioConSuTipo>",
    },
    Object {
      "@id": "http://inrupt.luispc1998/routeMedia/video/<comentarioConSuTipo>",
    },
  ],
  "name": "Route test 2",
  "points": Array [
    Object {
      "latitude": 45.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 46.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 47.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 48.123,
      "longitude": 32.121,
    },
    Object {
      "latitude": 49.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 40.123,
      "longitude": 32.121,
    },
    Object {
      "latitude": 50.123,
      "longitude": 33.121,
    },
    Object {
      "latitude": 53.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 54.123,
      "longitude": 34.121,
    },
    Object {
      "latitude": 55.123,
      "longitude": 35.121,
    },
    Object {
      "latitude": 57.123,
      "longitude": 30.121,
    },
    Object {
      "latitude": 56.123,
      "longitude": 26.121,
    },
    Object {
      "latitude": 53.123,
      "longitude": 25.121,
    },
    Object {
      "latitude": 55.123,
      "longitude": 20.121,
    },
    Object {
      "latitude": 51.123,
      "longitude": 15.121,
    },
  ],
}
`;
